x-pack/test/observability_ai_assistant_functional/tests/knowledge_base_management/index.spec.ts

```diff
@@ -197,5 +200,119 @@ export default function ApiTest({ getService, getPageObjects }: FtrProviderConte
         expect(savedContent).to.eql(originalInstruction);
       });
     });
+
+    describe('Bulk import functionality', () => {
+      const tempDir = os.tmpdir();
+      const tempFilePath = path.join(tempDir, 'bulk_import.ndjson');
+
+      async function prepareBulkImportData() {
+        const entries = [{"id":"12233","title":"Testing 4","text":"Contents of item"},
+        {"id":"6775","title":"Testing 2","text":"Contents of second item"},
+        {"id":"6573","title":"Testing","text":"Contents of third item"}]
```

Comment:
The formatting of this looks wrong and I would expect this to be caught by the linter. Do you have eslint and prettier running in your editor?